.github/workflows/publish-maven-package.yml

```diff
@@ -17,12 +17,18 @@ on:
   pull_request_target:
     types:
       - closed
+    branches:
```

Comment:
Instead of this, wouldn't `push` event on main branch make more sense ?

ref - https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push